roles/traefik/tasks/main.yml

```diff
@@ -8,6 +8,29 @@
     - traefik
     - bootstrap
 
+- name: create directory for local traefik service override
+  sudo: yes
+  file:
+    dest:  /etc/systemd/system/traefik.service.d
+    state: directory
+    mode: 0755
+  tags:
+    - traefik
+    - bootstrap
+
+- name: create local traefik service override
+  sudo: yes
+  copy:
+    dest: /etc/systemd/system/traefik.service.d/marathon.conf
+    content: |
+      [Service]
+      ExecStartPre=-/usr/bin/smlr {{ traefik_marathon_endpoint }}/v2/leader
```

Comment:
good question. I did that b/c I didn't want the unit and the other ExecStartPre commands to fail if smlr timed out or failed. 

```
ExecStartPre=/bin/touch /var/log/{{.Name}}/access.log
ExecStartPre=/bin/touch /var/log/{{.Name}}/traefik.log
```
(see https://github.com/asteris-llc/mantl-packaging/blob/master/packages/traefik/traefik.service)

In that case, traefik would still start but it would likely be in a bad state (not talking to marathon - the scenario in #1073). 

You could argue that it would be better if we remove the `-` so that if smlr fails, the unit fails and traefik won't be started at all. But, since smlr is very new, I thought it would be a little safer to use `-` so that if it fails for any other reason than a timeout, traefik would still get started. In both cases, the consul health checks should alert.

What do you think?